repo/packages/S/spark-history/0/package.json

```diff
@@ -20,6 +20,6 @@
         "analytics"
     ],
     "website": "https://docs.mesosphere.com/service-docs/spark/",
-    "version": "2.1.0-1",
-    "minDcosReleaseVersion": "1.7"
+    "version": "1.1.0-2.1.1",
```

Comment:
Can we keep this using the old scheme? `2.1.1-1.1.0` when we start using semver for `version` it will sort better. E.g. `2.1.1-1.1.0` > `2.0.10-1.2.0` as oppose to `1.2.0-2.0.10` > `1.1.0-2.1.1` which I think is not what the user expects.